This is workaround for a bug in tox 4.21.0.
We should drop this once newer tox is built and installed.

https://github.com/tox-dev/tox/issues/3386

--- tox-4.21.2/tox.toml.orig
+++ tox-4.21.2/tox.toml
@@ -1,4 +1,3 @@
-requires = ["tox>=4.19"]
 env_list = ["fix", "3.13", "3.12", "3.11", "3.10", "3.9", "3.8", "cov", "type", "docs", "pkg_meta"]
 skip_missing_interpreters = true
 
